torch/functional.py

```diff
@@ -98,28 +98,38 @@ def lu_unpack(LU_data, LU_pivots, unpack_data=True, unpack_pivots=True):
         >>> # can recover A from factorization
         >>> A_ = torch.bmm(P, torch.bmm(A_L, A_U))
```

Comment:
It would be nice to have an example for rectangular `A`.

Reply:
Yes. Added.